### ISSUE TITLE
Check for superuser should come first

### ISSUE THREAD

**Issue by Markus-N** [archlinuxfr/yaourt#13]
As an Arch newbie, I sometimes forget to use yaourt as normal user, because all other adminstrative tasks are done as root.
When using yaourt as root, it warns me about that, but it has already done some things before the warning.
I think, the check for superuser and the corresponding warning should be the very first thing, before any other task is performed. A "would you like to continue" confirmation would also be nice.
